src/app/components/game/interface/menu-bar/home-register/home-register.component.html

```diff
@@ -1,71 +1,36 @@
-        <img [src]="profileSource" alt="'Profile'">
-      </div>
-      <div (click)="fileInput.click()" class="upload-icon"><i class="fas fa-file-upload"></i></div>
-    </div>
-
-    <div class="info-section">
-      <div>
-        <div class="user-name">{{myPlayer?.displayName || 'Not connected'}}</div>
-        <div class="sub-script">Role: {{getRole()}}</div>
-        <div class="sub-script">Time played: {{getTimePlayed()}}</div>
-      </div>
-    </div>
-
-    <div class="bottle-cap-section">
-      <div (click)="prevBCap($event)"><i class="fas fa-chevron-left"></i></div>
-      <div class="round-avatar golden-border bottle-cap">
-        <div class="bottle-cap-cover">
-          <img [src]="bottleCapSource" alt="'Profile'">
-        </div>
-      </div>
-      <div (click)="nextBCap($event)"><i class="fas fa-chevron-right"></i></div>
-    </div>
-
-  </div>
-
-  <div class="seperator"></div>
-
   <app-trink-buddy-display></app-trink-buddy-display>
 
   <div class="seperator"></div>
 
   <div class="chat-box">
-
     <div #textSection class="text-section">
-      <div *ngFor="let msg of chatMessages" [ngClass]="{'move-right': msg.sender === myPlayer?.displayName}">
+      <div *ngFor="let msg of chatMessages" [ngClass]="{ 'move-right': msg.sender === myPlayer?.displayName }">
         <div class="chat-message">
           <div class="head-row">
-            <div class="sender"><i class="fas fa-user"></i>{{msg.sender}}</div>
-            <div class="time">{{msg.timestamp | date: 'shortTime'}}</div>
+            <div class="sender"><i class="fas fa-user"></i>{{ msg.sender }}</div>
+            <div class="time">{{ msg.timestamp | date: 'shortTime' }}</div>
           </div>
-          <div class="body chat-message-text">{{msg.text}}</div>
+          <div class="body chat-message-text">{{ msg.text }}</div>
         </div>
       </div>
     </div>
 
-
     <div class="seperator"></div>
 
     <div class="input-section">
       <div class="command-hint" (click)="showChatCmdDropdown = !showChatCmdDropdown">
         <i class="fas fa-terminal"></i>
-        <app-chat-command-list *ngIf="showChatCmdDropdown" (selectCmdEmitter)="selectCmd($event)"></app-chat-command-list>
+        <app-chat-command-list
+          *ngIf="showChatCmdDropdown"
+          (selectCmdEmitter)="selectCmd($event)"
+        ></app-chat-command-list>
       </div>
       <div (keyup)="sendChatMessageByKey($event, chatInput)" class="text-input-container">
-        <input #chatInput placeholder="Type here ..." type="text" spellcheck="false">
+        <input #chatInput placeholder="Type here ..." type="text" spellcheck="false" />
       </div>
       <div (click)="sendChatMessage(chatInput)" class="post-button"><i class="fas fa-share"></i></div>
     </div>
-
   </div>
-  <input
-    #fileInput
-    (change)="newProfilePic($event)" style="display: none"
-    type="file">
+  <input #fileInput (change)="newProfilePic($event)" style="display: none" type="file" />
```

Comment:
weird.. mal wirds so formatiert und mal wirds in multiline aufgeteilt. Nichts für diesen PR, aber evtl sollten wir uns mal die Format Richtlinien anschauen. Ist ja nicht Sinn der Sache hin und her zu formatieren, wenn die Zeile 1 Zeichen länger wird..

Reply:
wahrscheinlich noch iwelche configs in der IDE die wir nicht in unseren projektsettings definiert haben. dann wird immer IDE-Settings für die gegebene Rule genommen.